src/DynamicNLPModels.jl

```diff
@@ -14,11 +14,11 @@ abstract type AbstractLQDynData{T,S} end
 
 A struct to represent the features of the optimization problem 
 
-    minimize    1/2 sum(si^T Q si for i in 1:(N-1)) + 1/2 sum(ui^T R ui for i in 1:(N-1)) + 1/2 sN^T Qf s_N
-    subject to  s{i+1} = A s{i} + B u{i}  for i in 1:(N-1)
+    minimize    1/2 sum(si^T Q si for i in 0:(N-1)) + 1/2 sum(ui^T R ui for i in 0:(N-1)) + 1/2 sN^T Qf s_N
```

Comment:
I would suggest adding all math expressions inside `math` quotes, and write proper latex code. That way we can compile the expression nicely when building the doc.

Reply:
What constitutes `math` quotes? Does that just mean using the double \`\` symbols rather than the $ symbol in the doc string? I am going off of the following site in making that guess: https://juliadocs.github.io/Documenter.jl/stable/man/latex/